spec/system/claims/view_organisations_spec.rb

```diff
@@ -37,7 +37,7 @@ def and_persona_has_multiple_schools(persona)
   end
 
   def and_persona_has_one_school(persona)
-    create(:membership, user: persona, organisation: create(:school))
+    create(:membership, user: persona, organisation: create(:school, name: "Hogwarts"))
```

Comment:
I'm glad we didn't lose our Harry Potter reference. It lives on! 😆

<img src="https://pm1.narvii.com/6401/8c9acb4cd89b0942b930295ec05a143d0456ff98_hq.jpg" width="100" height="100" />